apps/storybook/src/Badge.test.ts

```diff
@@ -0,0 +1,15 @@
+describe('Badge', () => {
+  const storyPath = 'Core/Badge';
+  const tests = ['Basic', 'Long Label', 'Statuses'];
+
+  tests.forEach((testName) => {
```

Comment:
see https://github.com/iTwin/iTwinUI-react/pull/660#discussion_r869272468

do you have a more specific suggestion of the best way to do this? i want to avoid having to import the helper function in each file

Reply:
Why it is bad to import some helper versus having to copy paste everything and if we want to adjust something there, the good luck?

Reply:
Added `storyId` helper function into the `Cypress` object